### DISCUSSION TITLE
Make the tablet area position draggable

### DISCUSSION THREAD

**Discussion by J4C0B3Y** [ppy/osu#21913] [Ideas] [answered]
Make it so you can reposition the tablet area with your mouse rather then only using the `X Offset` and `Y Offset` sliders.

<img src="https://user-images.githubusercontent.com/69947689/209894804-9b2b0ac4-412a-4c7e-b801-8885fd2e848c.png" height=400>

This would be a nice QoL addition as its easier just to drag it with your mouse

**Comment by ggliv** [comment, marked as answer]
Implemented in #22697.
